dsls/pom.xml

```diff
@@ -55,14 +55,6 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-jar-plugin</artifactId>
-          <executions>
```

Comment:
With this removed, I think this whole `<plugin>` section does nothing, so you can just remove it.